### PR TITLE
docs(components/molecule/avatar): mod demo

### PR DESCRIPTION
**Pull request by andresin87**
## MOLECULE/AVATAR
**TASK**: 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Only
- [ ] Refactor

### CODE REVIEW

File: components/molecule/avatar/src/index.js [hunk 1]
```diff
@@ -11,68 +17,67 @@ import AvatarBadge, {
...
+      src,
+      fallbackIcon,
+      style,
+      isLoading,
+      children: childrenProp,
+      ...others
+    },
+    forwardedRef
+  ) => {
+    const baseClassName = 'sui-MoleculeAvatar'
```
Comment (nistadev) [inline components/molecule/avatar/src/index.js]: This can be moved now to `settings.js` file.

Comment (andresin87) [reply components/molecule/avatar/src/index.js]: 👍

Comment (andresin87) [reply components/molecule/avatar/src/index.js]: done

**Review by nistadev — APPROVED** [review]

**Review by turolopezsanabria — APPROVED** [review]
👏🏼  bravou

**Pull request merged**